### ISSUE TITLE
return predicted value for classification

### ISSUE THREAD

**Issue by samanemami** [zzd1992/GBDTMO#8]
Dear @zzd1992 

First of all, I have to thank you for all of your supports.

I noticed that the predict method, returns the probability of each class in size of n_class.
```Python
def predict(self, x, num_trees=0):
        preds = np.full((len(x), self.out_dim), self.base_score, dtype='float64')
        self.lib.Predict.argtypes = [ctypes.c_void_p, array_2d_double, array_2d_double,
                                     ctypes.c_int, ctypes.c_int]
        self.lib.Predict(self._boostnode, x, preds, len(x), num_trees)
        return preds
```
Is there any method to return the predicted value?

As with the probability, we can not continue with other metrics of classification to see the performance of the model.

**Comment by samanemami** [comment]
Or how can I return the loss to use it to convert the values?

**Comment by zzd1992** [comment]
If you use eval set, it computes and shows the acc of eval set during training. 

For test, you can compute acc based on the logits.

This project only implements the core algorithm. So you need to do things on your own. Sorry about this.

**Comment by samanemami** [comment]
Thank you.
I am trying to modify some parts based on my needs.
Is there any way to return the loss function after the training, for instance;
```Python
params = {"max_depth": 5, "lr": 0.1, 'loss': b"ce", 'verbose': False}
model = GBDTMulti(LIB, out_dim=3, params=params)
model.set_data((x_train, y_train), (x_test, y_test))
model.train(30)
model.lib['loss']
```

**Comment by samanemami** [comment]
Thanks for your reply @zzd1992 .
I modified it as the [link](https://github.com/samanemami/GBDTMO/blob/master/gbdtmo/gbdtmo.py). It would be great to have your confirmation.
